src/plugins/hotkeyPlugin.ts

```diff
@@ -34,6 +34,8 @@ export const useHotkeyManager = () => {
 
 type Editor = "talk" | "song";
 
+type BindingKey = string & { __brand: "BindingKey" };
```

Comment:
zodを使わずにブランド型を成り立たせたい感じですかね :eyes:
わかりづらい気がするので、コード行数を増やしてもいいからzodを導入するか、コメントを書いた方がいいかなと思いました...!

Reply:
確かに！　コメントを書き足そうと思います！
（zodの役割は`schema validation`なのでBrand型にするだけならこっちかなと思った次第です）

```suggestion
type BindingKey = string & { __brand: "BindingKey" }; // BindingKey専用のブランド型
```